artemis/reporting/modules/api_scanner/template_api_vulnerability.jinja2

```diff
@@ -0,0 +1,52 @@
+{% if "api_vulnerability" in data.contains_type %}
+    <li>{% trans %}We identified potential API security vulnerabilities:{% endtrans %}
+        <ul>
+            {% for report in data.reports %}
+                {% if report.report_type == "api_vulnerability" %}
+                    <li>
+                        {{ report.target }}
+                        <ul>
+                            <li>{% trans %}Method:{% endtrans %} {{ report.additional_data.method|upper }}</li>
+                            <li>{% trans %}Endpoint:{% endtrans %} {{ report.additional_data.endpoint }}</li>
+                            <li>{% trans %}Status Code:{% endtrans %} {{ report.additional_data.status_code }}</li>
+                            <li>
+                                {% trans %}Details:{% endtrans %}
+                                {% set raw_detail = report.additional_data.details %}
+                                {% if "HTTP verb" in raw_detail %}
+                                    {% trans %}Unsupported HTTP Method - The endpoint accepts HTTP methods that should not be allowed, potentially enabling unintended operations.{% endtrans %}
+                                {% elif "vulnerable to SQL Injection" in raw_detail or "vulnerable to SQli" in raw_detail %}
+                                    {% trans %}SQL Injection - Malicious input can manipulate database queries, leading to data leaks or unauthorized access.{% endtrans %}
+                                {% elif "time-based SQL Injection" in raw_detail %}
+                                    {% trans %}Time-based SQL Injection - An attacker can infer data using timing-based payloads, exploiting database behavior.{% endtrans %}
+                                {% elif "OS Command Injection" in raw_detail %}
+                                    {% trans %}OS Command Injection - Arbitrary system commands can be executed on the server, leading to full compromise.{% endtrans %}
+                                {% elif "XSS/HTML Injection" in raw_detail %}
+                                    {% trans %}XSS/HTML Injection - Untrusted input is rendered in the browser, enabling session hijacking or defacement.{% endtrans %}
```

Comment:
I think we may add a sentence or two describing the mechanism of defacement - why reflected xss leads to defacement ;)